### PR TITLE
Fix log output dir

### PR DESCRIPTION
**Pull request by rupal-bq**
*Issue #48*

*Description of changes:*
- Updates the log output directory
- The connection string has the highest preference.
- If the connection string doesn't have a log path then the value is taken from DSN.
- When connection string & DSN, don't have value for log path then the default value is used.
- Tested with Integration tests
- Tested with Tableau DSN & Driver options
- Administrative privileges are required to change the value of logging options.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,6 +57,8 @@ The following are the driver options that can be configured using a DSN or conne
 | `LogLevel` | Severity level for driver logs. | one of `ES_OFF`, `ES_FATAL`, `ES_ERROR`, `ES_INFO`, `ES_DEBUG`, `ES_TRACE`, `ES_ALL` | `ES_WARNING` |
 | `LogOutput` | Location for storing driver logs. | string | WIN: `C:\`, MAC: `/tmp` |
 
+Administrative privileges are required to change the value of logging options.
```
Comment (jordanw-bq) [inline README.md]: Add a bold **NOTE** here to highlight this point.

File: src/elasticodbc/drvconn.c [hunk 2]
```diff
@@ -105,7 +105,6 @@ dconn_DoDialog(HWND hwnd, ConnInfo *ci) {
 
 LRESULT CALLBACK dconn_FDriverConnectProc(HWND hdlg, UINT wMsg, WPARAM wParam,
                                           LPARAM lParam) {
-    MYLOG(ES_DEBUG, "dconn_FDriverConnectProc\n");
```
Comment (jordanw-bq) [inline src/elasticodbc/drvconn.c]: Restore this debug log line

File: src/elasticodbc/es_driver_connect.cpp [hunk 3]
```diff
@@ -179,6 +179,10 @@ static SQLRETURN SetupConnString(const SQLCHAR *conn_str_in,
         return SQL_ERROR;
     }
 
+    //This will be used to restore the log output dir fetched from connection string
+    //Since getDSNinfo overrides all available connection attributes
+    std::string conn_log_dir(ci->drivers.output_dir);
```
Comment (jordanw-bq) [inline src/elasticodbc/es_driver_connect.cpp]: Can we name this to `conn_string_log_dir` or something similar to help highlight the point that this is from the connection string?

File: README.md (hunk 1 above)
Comment (rupal-bq) [reply README.md]: added

File: src/elasticodbc/es_driver_connect.cpp (hunk 3 above)
Comment (rupal-bq) [reply src/elasticodbc/es_driver_connect.cpp]: renamed

File: src/elasticodbc/drvconn.c (hunk 2 above)
Comment (rupal-bq) [reply src/elasticodbc/drvconn.c]: restored

**Pull request merged**